runtime/near-test-contracts/Cargo.toml

```diff
@@ -11,3 +11,6 @@ license = "Apache-2.0"
 [dependencies]
 once_cell = "1"
 wat = "1.0.40"
+wasm-smith = "0.10.0"
+rand = { version = "0.8", features = ["small_rng"] }
+arbitrary = "1.1"
```

Comment:
Very minor, but I now think it's good to specify patch version for dep as well https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277